dist/app.js.LICENSE.txt

```diff
@@ -0,0 +1,6 @@
+/* @license
```

Comment:
Interesting @bgrgicak! I checked and it looks like the CSV Validator module I used has papaparse as a dependency - https://github.com/kdevnel/ynab-absa-csv-convert-js/blob/52a47c23719a2ae183d46228c879c4ff3616d7ca/package-lock.json#L1559

I can only assume that package emits a license file when you use a production environment since the license file seems to contain the license info for that package.